### DISCUSSION TITLE
Order by is case sensitive

### DISCUSSION THREAD

**Discussion by rakeshkhoodeeram** [supabase/supabase#2402] [Questions] [answered]
When I call https://xhabqccfdobmicajnvqx.supabase.co/rest/v1/s_users_lists?order=name
I had items being  fetched in order as: Ration, dwdwd, njn, which is not correct as it had to be: dwdwd, njn, Ration.
When I changed them to upper case, then I got, DWDWD, NJN, RATION.

**Comment by steve-chavez** [comment, marked as answer]
To override the default behavior, you can lowercase the names by using a [computed column](https://postgrest.org/en/stable/api.html#computed-columns). Like:

```sql
create function lower_name(s_users_lists s) returns text as $$
  select lower(s.name);
$$ language sql;
```

Then doing:

```
/rest/v1/s_users_lists?order=lower_name
```

Should work as you expect.

**Reply by taylorhlabs** [reply]
@steve-chavez How can this be implemented using `.order()` in the Supabase client API?

**Reply by steve-chavez** [reply]
Should be

```js
const { data, error } = await supabase
  .from('s_users_lists')
  .select('*')
  .order('lower_name')
```
